src/service.ts

```diff
@@ -242,10 +242,14 @@ export class DebuggerService implements IDebugger, IDisposable {
       await this.start();
     }
 
+    if (this.isStarted || autoStart) {
+      this._model.title = this.isStarted ? this.session?.connection?.name : '-';
+    }
+
     this._model.breakpoints.restoreBreakpoints(bpMap);
     if (stoppedThreads.size !== 0) {
       await this._getAllFrames();
-    } else {
+    } else if (this.isStarted) {
```

Comment:
Should we leave this change out of this PR for now?

I believe this is to handle #351, but it looks like other changes would be required too.